core/templates/core/index.html

```diff
@@ -22,25 +22,25 @@
 </script>
 {% endblock %}
 
-
 {% block content %}
+<div class="h-screen w-screen">
+    <nav class="flex flex-wrap justify-between items-center mx-auto w-screen p-2 border-gray-800 bg-gradient-to-b from-indigo-700 via-indigo-800 to-indigo-900">
```

Comment:
```suggestion
    <nav class="flex flex-wrap justify-between items-center w-screen p-2 border-gray-800 bg-gradient-to-b from-indigo-700 via-indigo-800 to-indigo-900">
```
By default, all elements have margin auto

Reply:
its just what i copied form tailwind i have no idea what it does :33